tests/solver_history/screen-output

```diff
@@ -0,0 +1,109 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 1.4.pre
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+--     . using Trilinos
+-----------------------------------------------------------------------------
+
+Loading shared library <./libsolver_history.so>
+
+Connecting signals
+Number of active cells: 1,024 (on 6 levels)
+Number of degrees of freedom: 13,764 (8,450+1,089+4,225)
+
+*** Timestep 0:  t=0 seconds
+   Solving temperature system... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 
+post_stokes_solver:
+4.30549e-07
+4.29777e-07
+4.29207e-07
+4.26588e-07
+4.15948e-07
+3.95221e-07
+3.7013e-07
+3.13502e-07
+2.15859e-07
+1.53156e-07
+1.18936e-07
+8.33093e-08
+5.01955e-08
+3.1172e-08
+2.09542e-08
+1.53074e-08
+1.10199e-08
+6.96632e-09
+4.10842e-09
+2.65276e-09
+1.87224e-09
+1.34178e-09
+9.4782e-10
+6.03066e-10
+3.71778e-10
+2.33403e-10
+1.51829e-10
+1.06127e-10
+7.63826e-11
+4.97925e-11
+4.97925e-11
+3.69443e-11
+-1
+3.69443e-11
+2.40324e-11
```

Comment:
I guess the `-1` here indicates a restart iteration? Do we document this in the description of the signal?